src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -6,8 +6,12 @@
 <%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
 <%@ taglib tagdir="/WEB-INF/tags/instructor/courseStudentDetails" prefix="ticsd" %>
 <c:set var="jsIncludes">
+    <script type="text/javascript" src="<%= FrontEndLibrary.TINYMCE %>"></script>
+    <script type="text/javascript" src="/js/richTextEditor.js"></script>
     <script type="text/javascript" src="/js/instructor.js"></script>
     <script type="text/javascript" src="/js/instructorStudentRecords.js"></script>
+    <script type="text/javascript" src="/js/feedbackResponseComments.js"></script>
+    <script type="text/javascript" src="/js/instructorComments.js"></script>
```

Comment:
This file should no longer exist. This could be why you get the false positive in your own dev server.